measures/ResStockArguments/tests/resstock_arguments_test.rb

```diff
@@ -35,4 +39,125 @@ def test_options_lookup_assignment
     puts "resstock_arguments - lookup_arguments: #{resstock_arguments_extras.sort}" if !resstock_arguments_extras.empty?
     assert_equal(0, resstock_arguments_extras.size)
   end
+
+  def test_create_geometry_envelope
+    characteristics_dir = File.join(File.dirname(__FILE__), '../../../project_national/housing_characteristics')
+    parameters_ordered = get_parameters_ordered_from_options_lookup_tsv(@lookup_csv_data, characteristics_dir)
+    measures_dir = File.join(File.dirname(__FILE__), '../../../measures')
+    parameter_options_measure_args = _get_parameter_options_measure_args(@lookup_csv_data)
```

Comment:
Do this operate once ahead of the loop through 10K datapoints -- much faster than calling `get_measure_args_from_option_names` for every datapoint.